src/main/java/seedu/address/model/user/UserSession.java

```diff
@@ -0,0 +1,61 @@
+package seedu.address.model.user;
```

Comment:
Isn't this file already added in another PR #33?

Reply:
yep. should we'll relook at this PR after #33 is merged. @kohchihao please *rebase* later 😉.